cameraserver/src/main/java/edu/wpi/first/cameraserver/CameraServerShared.java

```diff
@@ -43,4 +43,13 @@
    * @param id the usage id
    */
   void reportAxisCamera(int id);
+
+  /**
+   * Get if running on a roboRIO.
```

Comment:
```suggestion
   * Gets if running on a roboRIO.
```

Reply:
All the other ones in this file are in the format I used. I'd rather have the format match, and the format I used is clearer and makes more sense anyway. So I'll keep this, and a separate PR can be done to change it if you want.